<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Data_collect">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Data_collect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start_up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Init">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialization</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>410</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>440</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>530</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>500</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>530</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sockfd_PLC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sockfd_MONITOR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Communication_PLC">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Communication_PLC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Communication_MONITOR">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Communication_MONITOR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_control_0">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_control_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_control_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_control_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_control_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_control_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>190</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_control_6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>190</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_monitor_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_monitor_6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>260</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_monitor_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_monitor_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>260</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_monitor_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>260</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_monitor_0">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DP_monitor_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>DP_control</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>DP_monitor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Speed_up">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed_up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Speed_down">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed_down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Epos_up">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Epos_up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Epos_down">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Epos_down</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
